### DISCUSSION TITLE
🚀Good news, folks! Mixcore v1.0.3 is now released

### DISCUSSION THREAD

**Discussion by Smilefounder** [mixcore/mix.core#481] [Announcements]
### 
After released Mixcore v1.0.1. We have used this version to develop some new website projects for our friends and clients. Anything working perfectly with only some minor bugs found.

Based on our actual projects, we did improve a lot of features here and there to make Mixcore better and better day by day.

And then today, I and my team are happy to let you know that Mixcore v1.0.3 is now released with a beautiful admin portal, more cleaner and more relaxed.

![Mixcore Admin Portal Bootstrap 5.x](https://user-images.githubusercontent.com/3785721/125382627-322d7d00-e3c0-11eb-8ff7-f02316770876.png "Mixcore CMS Admin Portal Bootstrap 5")

The download links are below:

- Release note: https://github.com/mixcore/mix.core/releases/tag/v1.0.3
- Download (49Mb): https://github.com/mixcore/mix.core/releases/download/v1.0.3/mixcore-v1.0.3.zip

And please always remember that we are happy to support you anytime if you facing any issues or concerns.

Best regards,
Huy from Mixcore Team

### Release notes:

- Fix Migration issues
- Fix Export Data issues

### Notes:

- Do remember to clear browser cache / application cache

<hr /><em>This discussion was created from the release <a href='https://github.com/mixcore/mix.core/releases/tag/v1.0.3'>Mixcore v1.0.3</a>.</em>

**Comment by biapar** [comment]
Is there a showcases page?

**Reply by Smilefounder** [reply]
Hi @biapar, you can access some websites below:
- https://haiyeneyecare.com & https://matansinh.com (this is from 2 years ago with Mixcore beta version & don't core v3)
- https://wineme.today (latest version)
- https://dev.mixcore.org
- https://sjtc.ru
- http://54.159.73.158:8081 
- some other sites...

**Comment by droivandam** [comment]
Hi!
Can you tell how to upgrade from 1.0.1 to 1.0.3? I installed and configured the site, I do not want to lose progress.

**Reply by Smilefounder** [reply]
@nhathoang989 kindly advise @droivandam on this.

**Reply by droivandam** [reply]
I think 1.0.3 got memore leak. On my hosting is less stable then 1.0.1. When setting up the landing page, the server crashes. Tell me which version is the most stable and without bugs?

**Reply by Smilefounder** [reply]
@droivandam is it possible to share with us access to your site? So we can double-check. As v1.0.3 is not changed much compared to v1.0.1.

Btw, FYI. We used mochahost (cheapest shared hosting package) still able to run v1.0.3 properly.
